test/congr.lean

```diff
@@ -33,19 +33,24 @@ example {α β : Type _} {F : _ → β} {f g : { f : α → β // f = f }}
   guard_target = type_of% h
   exact h
 
-private opaque List.sum : List Nat → Nat
+-- Adaptation note: the next two examples have always failed if `List.ext` was in scope,
+-- but until nightly-2024-04-24 (when `List.ext` was upstreamed), it wasn't in scope.
```

Comment:
Is it possible to `attribute [-ext] List.ext` here? Alternatively, use a similar but different example.

Reply:
Done.